lib/config-generator.js

```diff
@@ -383,6 +395,11 @@ ConfigGeneraror.prototype = {
                     } else {
                         moduleName = node.arguments[0].value;
                         dependencies = node.arguments[1];
+
+                        // Save the file back
+                        if (!self._options.skipFileOverride) {
+                            self._saveFile(file, ast);
+                        }
```

Comment:
If this is the case, all the `define` branches require a `saveFile` (because of the possible `namespace` rewrite, so we can simply save once for all branches at the end.